.dockerignore

```diff
@@ -7,3 +7,6 @@ bin/release
 circle.yml
 pkg
 tags
+*.swp
```

Comment:
SGTM - I know `.swf` is used for flash animations, so as long as we don't intend to include those in the future, we should be good